superset-frontend/src/components/Table/Table.stories.tsx

```diff
@@ -35,6 +35,13 @@ export default {
   title: 'Design System/Components/Table/Examples',
   component: Table,
   argTypes: { onClick: { action: 'clicked' } },
+  decorators: [
+    Story => (
+      <ThemeProvider theme={supersetTheme}>
+        <Story />
+      </ThemeProvider>
+    ),
+  ],
```

Comment:
The theme is already provided [here](https://github.com/apache/superset/blob/394fb2f2d0e05f27ced88e8ff4fc6994696cab68/superset-frontend/.storybook/preview.jsx#L50).
```suggestion
```

Reply:
great, nice catch!